src/Oro/Bundle/ActionBundle/Resources/doc/operations.md

```diff
@@ -121,10 +121,10 @@ operations:
                 attribute: $.date                                   # action parameters
 
         preconditions:                                              # (optional) pre conditions for display Action button
-            @gt: [$updatedAt, $.date]                               # condition definition
+            - "@gt: [$updatedAt, $.date]"                               # condition definition
```

Comment:
Hi @mchojrin ,
We prefer single quotation syntax for the yaml files, and it is required to quote only `@gt` expression, not the whole string.
```suggestion
            `@gt`: [$updatedAt, $.date]                             # condition definition